share/spack/gitlab/cloud_pipelines/stacks/radiuss-aws-aarch64/spack.yaml

```diff
@@ -0,0 +1,161 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: false
+
+  config:
+    concretizer: clingo
+    install_tree:
+      root: /home/software/spack
+      padded_length: 384
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  packages:
+    all:
+      providers:
+        blas:
+          - openblas
+        mkl:
+          - intel-oneapi-mkl
+        mpi:
+          - openmpi
+          - mpich
+      variants: +mpi
+
+  definitions:
+  - radiuss:
+    - ascent
+    - blt
+    - caliper
+    - caliper +cuda cuda_arch=70
+    - camp
+    - camp +cuda
+    - chai
+    - chai +cuda +raja
+    - mfem
+    - mfem +superlu-dist+petsc+sundials
+    - mfem +cuda cuda_arch=70
```

Comment:
In this spec, it is better to also add `^hypre+cuda` (no need to add `cuda_arch` after this since the `mfem` package propagates it to `hypre`).